bridge/server/user/oauth.ts

```diff
@@ -47,6 +54,13 @@ async function setupClient(discoveryEndpoint: string, clientId: string, redirect
     console.log(
       `RP logout is supported by OAuth service. Using logout endpoint : ${ssoIssuer.metadata.end_session_endpoint}.`
     );
+    if (defaultContentSecurityPolicyOptions.directives) {
+      defaultContentSecurityPolicyOptions.directives['form-action'] = [
+        `'self'`,
+        `${ssoIssuer.metadata.end_session_endpoint}`,
+      ];
+      app.use(helmet.contentSecurityPolicy(defaultContentSecurityPolicyOptions));
+    }
```

Comment:
I would expect this to be outside of the setup of the client function and more a concern of `setupOAuth()`.